### ISSUE TITLE
Player Cash / Points System

### ISSUE THREAD

**Issue by Igneous01** [KaukausInsurgency/kaukasusinsurgency-dcs#202]
With the discussions about adding airdrop requests and combat convoys, one question that was unanswered was how to prevent players from spamming or abusing the system.

One way would be to implement a cash/points system for players, so that if they want to request the airdrop/convoy they need to pay for it.

Cash would be earned from doing things in game - Taking off, Landing, Total Flight Time, Kills, Transport Mount and Dismount, crate sling hook and unhook, depot resupplies, crate unpacking, etc.

What sucks is that we have to implement the same code that we use for stats tracking into the main DCS Scripts themselves, but this at least works around the problem of abuse, and opens up lots of potential for players to be rewarded for their efforts.

Would need to store player cash either in the player record itself (so cash carries over from any server) or, restrict cash to per server (so you cant use money earned in a different server to call in stuff). Would need feedback from server owners about how they feel about this.

**Comment by Frosties82** [comment]
I Think it is a good idea for a public server, but the option to disable it would also be important for a Closed Group to use.

**Comment by Igneous01** [comment]
Yes that's an important point to bring up - not everyone would want to use cash/point system especially if running private campaign or mission with group.

**Comment by thebgpikester** [comment]
Also, the current time-out system is quite effective, again, still for closed group, but I think the design goal should be to get regular players something to benefit their return to the server. 

A very simple rendition would be to use the MOOSE points system that happens automatically and deduct from it. So for every 10 units killed, they can command a paradrop and reset their balance.
